incident-bot.json

```diff
@@ -341,17 +449,21 @@
    },
    "WaitCondition" : {
      "Type" : "AWS::CloudFormation::WaitCondition",
-     "DependsOn" : "IncidentBotInstance",
+     "DependsOn" : "BotInstance",
      "Properties" : {
          "Handle" : { "Ref" : "BotWaitHandle" },
          "Timeout" : "900"
       }
     }
   },
   "Outputs": {
```

Comment:
At the end of a CFN run, the is an outputs tab. This is where you can build a quick reference sheet for users to grab information about a stack.